IPython/terminal/interactiveshell.py

```diff
@@ -69,26 +87,24 @@ def get_default_editor():
 
     if os.name == 'posix':
         return 'vi'  # the only one guaranteed to be there!
-    else:
-        return 'notepad' # same in Windows!
+    return 'notepad' # same in Windows!
 
 # conservatively check for tty
 # overridden streams can result in things like:
 # - sys.stdin = None
 # - no isatty method
 for _name in ('stdin', 'stdout', 'stderr'):
     _stream = getattr(sys, _name)
+    _is_tty = True
```

Comment:
you can actually move that out of the for loop if you really want.

Reply:
Yea, didn't notice that. Thanks.